lib/stripe/api.ex

```diff
@@ -376,14 +376,22 @@ defmodule Stripe.API do
   end
 
   defp do_perform_request_and_retry(method, url, headers, body, opts, {:attempts, attempts}) do
+    telemetry_meta = %{
+      endpoint: URI.parse(url).path,
```

Comment:
```suggestion
      http_url: URI.parse(url),
```

Pass the entire URL, let processors to filter the info, HTTP OTEL packages would do something around the line of https://github.com/open-telemetry/opentelemetry-erlang-contrib/blob/3f13bc8b2042071b723d0ea8c7b118fd10f42d26/instrumentation/opentelemetry_req/lib/opentelemetry_req.ex#L143-L148

Reply:
okay added and I renamed original field to `stripe_api_endpoint`. I can remove it because you're right that it can be extracted from `http_url` via processor, I kept it for better out-ot-the-box experience.